Fitness/Controllers/ClassDetailViewController.swift

```diff
@@ -225,47 +225,36 @@ extension ClassDetailViewController: UICollectionViewDataSource, UICollectionVie
         return CGSize(width: collectionView.frame.width, height: 360)
     }
 
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
+        return 0
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+        return 0
```

Comment:
i think default is 0?

Reply:
but need this one or else debug view hierarchy shows gaps b/w the cells that makes the height calculation off